src/tests/ftest/util/test_utils_pool.py

```diff
@@ -481,13 +482,35 @@ def wait_for_rebuild(self, to_start, interval=1):
                 waiting for rebuild to start or end
 
         """
+        if self.pool_query_timeout.value is not None:
+            self.log.info(
+                "Waiting for pool query to be responsive %s",
+                " with a {} second timeout".format(
+                    self.pool_query_timeout.value))
+
+            end_time = time() + self.pool_query_timeout.value
+            while time() < end_time:
+                try:
+                    self.dmg.pool_query(self.pool.get_uuid_str())
+                    self.log.info("Pool query still responsive")
+                    break
+                except CommandFailure as err:
+                    self.log.info("Pool Query still non-responsive {}".\
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments